src/m365/flow/commands/recyclebinitem/recyclebinitem-restore.ts

```diff
@@ -0,0 +1,62 @@
+import PowerAutomateCommand from '../../../base/PowerAutomateCommand.js';
+import { globalOptionsZod } from '../../../../Command.js';
+import { z } from 'zod';
+import { zod } from '../../../../utils/zod.js';
+import { Logger } from '../../../../cli/Logger.js';
+import commands from '../../commands.js';
+import { validation } from '../../../../utils/validation.js';
+import { formatting } from '../../../../utils/formatting.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+
+const options = globalOptionsZod
+  .extend({
+    environmentName: zod.alias('e', z.string()),
+    flowName: zod.alias('n', z.string()
+      .refine(name => validation.isValidGuid(name), name => ({
+        message: `'${name}' is not a valid GUID.`
```

Comment:
Let's add the option name. When you have multiple environments, the ID of the not-default environment is also a guid so this may cause confusion.

Reply:
When using ZOD, the failing option name is prefixed to the error message.